android/tangram/jni/platform_android.cpp

```diff
@@ -328,5 +337,11 @@ void initGLExtensions() {
     glGenVertexArraysOESEXT = (PFNGLGENVERTEXARRAYSOESPROC) dlsym(libhandle, "glGenVertexArraysOES");
 }
 
+std::string stringFromJString(JNIEnv* jniEnv, jstring string) {
+    const char* cstring = jniEnv->GetStringUTFChars(string, NULL);
```

Comment:
Updated all string conversions to use the helper function, which itself uses `GetStringUTFRegion`